style.css

```diff
@@ -1,468 +1,1180 @@
+  background-position-x: 647px;
+}
+
+.aside-second {
+  position: absolute;
+  margin-right: auto;
+  padding-bottom: 22px;
+  width: 440px;
+  background: #e51937;
+}
+
+.aside-second__title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  padding-top: 53px;
+  padding-left: 56px;
+  color: #fff;
+}
+
+.aside-second__content {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 19px;
+  padding-top: 11px;
+  padding-left: 56px;
+  width: 370px;
+  color: #fff;
+}
+
+.subtitle__text {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  padding-top: 55px;
+  padding-left: 56px;
+  color: #fff;
+}
+
+.aside-second__reting {
+  display: flex;
+  padding-left: 57px;
+  padding-top: 28px;
+}
+
+.reting__text {
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 21px;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.text__link:nth-child(2) {
+  padding-top: 17px;
+  padding-bottom: 15px;
+}
+
+.text__link:nth-child(3) {
+  display: flex;
+  padding-bottom: 17px;
+}
+
+.text__link:nth-child(4) {
+  padding-bottom: 16px;
+}
+
+.star__link:nth-child(1) {
+  margin-top: -2px;
+}
+
+.star__link:nth-child(2) {
+  padding-top: 9px;
+  padding-bottom: 10px;
+}
+
+.star__link:nth-child(4) {
+  padding-top: 8px;
+  padding-bottom: 9px;
+}
+
+.text__link__title {
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 12px;
+  text-transform: uppercase;
+  padding-top: 7px;
+  padding-left: 2px;
+  color: #fff;
+}
+
+.reting__star {
+  margin-left: 32px;
+  width: 160px;
+  justify-content: space-between;
 }
+
+.star__foto {
+  width: 25px;
+}
+
+.star__foto:nth-child(2) {
+  margin-left: -1px;
+  margin-right: -2px;
+}
+
+.star__foto:nth-child(4) {
+  margin-right: -4px;
+}
+
+.star__list,
+.text__list {
+  list-style: none;
+}
+
+.information__block__aside-five {
+  padding-top: 5px;
+}
+
+.aside-second__five {
+  display: flex;
+  padding-left: 64px;
+  padding-top: 92px;
+}
+
+.text__foto {
+  width: 65px;
+  height: 82px;
+  padding-top: 10px;
+}
+
+.number__title__aside-five,
+.number__subtitle__aside-five {
+  font-size: 50px;
+}
+
+.number__title__aside-five {
+  padding-left: 23px;
+}
+
+.number__subtitle__aside-five {
+  padding-top: 0;
+  padding-left: 13px;
+}
+
+.number__text__aside-five {
+  font-size: 25px;
+  margin-top: -9px;
+  padding-left: 24px;
+}
+
+.group__second__aside-five {
+  justify-content: center;
+  padding-right: 27px;
+  padding-top: 44px;
+}
+
+.section-third__car {
+  padding-top: 172px;
+  padding-left: 344px;
+}
+
+.car__foto {
+  position: relative;
+  width: 100%;
+}
+
 /*Адаптивное отображение*/
+
+@media (max-width: 1411px) {
+  .section-second {
+    background-size: 100%;
+  }
+}
+
+@media (max-width: 1300px) {
+  .section-second {
+    background-position: -5px 123px;
+  }
+}
+
+@media (max-width: 1251px) {
+  .section-second {
+    background-position: -5px 200px;
+  }
+}
+
+@media (max-width: 1121px) {
+  .content__img {
+    padding-right: 0;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)